### ISSUE TITLE
TypeError: Object doesn't support property or method 'find'

### ISSUE THREAD

**Issue by mohadib** [ryandrewjohnson/react-localize-redux#17]
Hello,

Looks like getActiveLanguage is expecting Array.find to exist. In IE11 this is causing an error. I created the app with create-react-app and have not ejected or customized anything. Here are some screenshots of the stacktrace and my code that sets up the store. Any ideas what I could do to fix this.. ? Thanks

![react1 error](https://user-images.githubusercontent.com/5350023/29475443-fa3a583e-841c-11e7-9018-77206668fc84.png)

![react2 error](https://user-images.githubusercontent.com/5350023/29475583-a4800bcc-841d-11e7-8559-fea70615b44b.png)

The code that sets up the store:

```javascript
import {applyMiddleware, createStore, combineReducers} from "redux";
import {localeReducer} from 'react-localize-redux';
import {createLogger} from "redux-logger";
import thunk from "redux-thunk"

const middleware = applyMiddleware(thunk, createLogger());

const reducers = combineReducers({
	locale: localeReducer
});

export default createStore(reducers, middleware);
```

**Comment by ryandrewjohnson** [comment]
Thanks for catching this.

Just pushed a fix for this in `v2.5.2`. Let me know if this fixes the issue, and if you come across any other IE specific issues.

**Comment by mohadib** [comment]
Just tied it out and problem is solved. Thanks.
